docs/sources/docker-hub-enterprise/release-notes.md

```diff
@@ -18,6 +18,70 @@ page_keywords: docker, documentation, about, technology, understanding, enterpri
 
 ## Commercialy Supported Docker Engine
 
+### CS Docker Engine 1.6.2-cs5
+
+For customers running Docker Engine on [supported versions of RedHat Enterprise
+Linux](https://www.docker.com/enterprise/support/) with [SELinux
+enabled](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/
+6/html/Security-Enhanced_Linux/sect-Security-Enhanced_Linux-Working_with_SELinux
```

Comment:
Looks like you wrapped the URL here; better check if that still works because of that